server/command.go

```diff
@@ -236,6 +258,102 @@ func showInstallCloudHelp(p *Plugin, context *model.CommandArgs, args ...string)
 	})
 }
 
+func addConfig(p *Plugin, context *model.CommandArgs, args ...string) *model.CommandResponse {
+	if !utils.IsSystemAdmin(context.UserId) {
+		p.postCommandResponse(context, installOnlySystemAdmin)
+		return &model.CommandResponse{}
+	}
+
+	elements := []model.DialogElement{
+		{
+			DisplayName: configServerURL,
+			Name:        configServerURL,
+			Type:        "text",
+			Placeholder: "https://example.com",
+			HelpText:    "Please enter your Confluence Server URL",
+			Optional:    false,
+		},
+		{
+			DisplayName: configClientID,
+			Name:        configClientID,
+			Type:        "text",
+			Placeholder: configClientID,
+			HelpText:    "Please enter your Confluence OAuth Client ID",
```

Comment:
```suggestion
			HelpText:    "Please enter your Confluence oauth client ID",
```
we should follow general rule keep only Noun(name) starting with capital letter